django_lightweight_queue/app_settings.py

```diff
@@ -21,3 +21,7 @@ def setting(suffix, default):
 REDIS_HOST = setting('REDIS_HOST', '127.0.0.1')
 REDIS_PORT = setting('REDIS_PORT', 6379)
 REDIS_PREFIX = setting('REDIS_PREFIX', '')
+
+ENABLE_PROMETHEUS = setting('ENABLE_PROMETHEUS', True)
```

Comment:
Backwards-compatible has quite a specific meaning - it means that any operation that succeeds on the previous version of the codebase won't fail on the succeeding one. I think that's true here.

It's naturally a shame if consumers of this library don't need prometheus, or get pwned because they haven't set up firewalls, but I don't think it's sensible to make that the overriding concern - we should be trying to make a library that is as easy for us to use as possible and hope that other people find it useful.